scripts/osx-sign.js

```diff
@@ -21,9 +19,9 @@ function checkMacPackageExistance() {
 
 function runOsxSign() {
```

Comment:
This is going to end up being dead code, as we are trying to get away from signing on local machines all together. How about we remove `runOsxSIgn` and its dependencies? What do you think?